Packs/BmcHelixRemedyForce/Scripts/BMCHelixRemedyforceCreateIncident/BMCHelixRemedyforceCreateIncident.py

```diff
@@ -0,0 +1,328 @@
+            using_argument
+        ),
+        'impact_id': get_field_id(
+            args.get("impact_id"),
+            args.get("impact"),
+            GET_COMMANDS_FOR_INCIDENT["impact_id"],
+            {"impact_name": args.get("impact")},
+            using_argument
+        ),
+        'account_id': get_field_id(
+            args.get("account_id"),
+            args.get("account"),
+            GET_COMMANDS_FOR_INCIDENT["account_id"],
+            {"account_name": args.get("account")},
+            using_argument
+        ),
+        'service_offering_id': get_field_id(
+            args.get("service_offering_id"),
+            args.get("service_offering"),
+            GET_COMMANDS_FOR_INCIDENT["service_offering_id"],
+            {"service_offering_name": args.get("service_offering")},
+            using_argument
+        ),
+        'template_id': get_field_id(
+            args.get("template_id"),
+            args.get("template"),
+            GET_COMMANDS_FOR_INCIDENT["template_id"],
+            {"template_name": args.get("template")},
+            using_argument
+        ),
+        'broadcast_id': get_field_id(
+            args.get("broadcast_id"),
+            args.get("broadcast"),
+            GET_COMMANDS_FOR_INCIDENT["broadcast_id"],
+            {"broadcast_name": args.get("broadcast")},
+            using_argument
+        ),
+        'asset_id': get_field_id(
+            args.get("asset_id"),
+            args.get("asset"),
+            GET_COMMANDS_FOR_INCIDENT["asset_id"],
+            {"asset_name": args.get("asset")},
+            using_argument
+        ),
+        'outage_start': args.get("outage_start"),
+        'outage_end': args.get("outage_end"),
+        'description': args.get("description"),
+        'opened_date': args.get("opened_date"),
+        'due_date': args.get("due_date")
+    }
+    command_args = generate_command_args_with_additional_fields(additional_fields)
+    command_args = remove_empty_elements(command_args)
+    if using_argument:
+        command_args["using"] = using_argument
+    command_res = demisto.executeCommand('bmc-remedy-incident-create', command_args)
+    result = {}
+    try:
+        entry = command_res[0]
+        context_output_type = find_entry_type_of_incident(demisto.get(entry, "Type"))
+        context_outputs = demisto.get(entry, "EntryContext")
+        human_readable = demisto.get(entry, "HumanReadable")
+        contents_format = demisto.get(entry, "ContentsFormat")
+        if isError(entry):
+            return_error(entry['Contents'])
+
+        else:
+            record_data = demisto.get(entry, 'Contents')
+            if not record_data:
+                return_error(ERROR_MESSAGES + json.dumps(entry))
+            else:
+                if demisto.get(record_data, "ErrorMessage"):
+                    result = record_data['ErrorMessage']
+                else:
+
+                    # Output entry
+                    result = {
```

Comment:
Use `CommandResults` https://xsoar.pan.dev/docs/integrations/context-and-outputs

Reply:
We have used result due to following reason-

- We have warning human readable and also normal human readable in create service request and create incident scripts.
- So we are directly sending command output to war room and not changing EntryType inside the script.
-  If we want to use CommandResults, we need to then check EntryType which is returned from actual command and put if else conditions based on EntryType and use return_results or return_warning based on EntryType

Are you expecting this kind of code handling?

Reply:
It can stay as it is then.